CHANGELOG.md

```diff
@@ -1,3 +1,21 @@
+# 4.0.0 (2020-05-21)
+
+### New features
+
+- Added support for `@angular/core@^9`. [#56](https://github.com/blackbaud/skyux-tabs/pull/56)
+- Updated the pipeline to transpile to the [Angular Package Format](https://docs.google.com/document/d/1CZC2rcpxffTDfRDs6p1cfbmKNLA6x5O-NtkJglDaBVs/preview). [#56](https://github.com/blackbaud/skyux-tabs/pull/56)
+
+### Breaking changes
+
+- Removed the `tabStyle` input from the tabset component, which was used to switch between "wizard" or "tabs" tab styles. If you wish to use a "wizard" styled component, use the progress indicator component. [#84](https://github.com/blackbaud/skyux-tabs/pull/84)
```

Comment:
Would wizard and tabs be better suited to ticks instead of quotation marks because those were values you set?

In the second sentence, I'd delete "If you wish" and start at "To use." Also swap "'wizard' styled" for "wizard-style".